src/iOS/Xamarin.iOS/Samples/GraphicsOverlay/SketchOnMap/SketchOnMap.cs

```diff
@@ -0,0 +1,350 @@
+﻿// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using Foundation;
+using System.Collections.Generic;
+using UIKit;
+using System;
+using System.Threading.Tasks;
+using System.Linq;
+using System.Drawing;
+
+namespace ArcGISRuntimeXamarin.Samples.SketchOnMap
+{
+    [Register("SketchOnMap")]
+    public class SketchOnMap : UIViewController
+    {
+        // Constant holding offset where the MapView control should start
+        private const int yPageOffset = 60;
+
+        // Create and hold reference to the used MapView
+        private MapView _myMapView = new MapView();
+
+        // Dictionary to hold sketch mode enum names and values
+        private Dictionary<string, int> _sketchModeDictionary;
+
+        // Graphics overlay to host sketch graphics
+        private GraphicsOverlay _sketchOverlay;
+
+        // Segmented control to show sketch controls
+        private UISegmentedControl _segmentButton;
+        
+        public SketchOnMap()
+        {
+            Title = "Sketch on map";
+        }
+
+        public override void ViewDidLoad()
```

Comment:
I added functionality today to allow iOS samples to rotate to adapt to device orientation. Having the CreateLayout() method call in the ViewDidLoad methods means that  when the device's orientation is changed the map will not change in size to fit the screen. 

Adding the CreateLayout() and Initialise() calls to a ViewDidLayoutSubviews() method instead will resolve this issue.

See https://devtopia.esri.com/runtime/dotnet-sdk/issues/1110 for details